### PR TITLE
Feature/fa 438

### PR DESCRIPTION
**Pull request by CH-JonathanBlake**
Created basic application

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+                      http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <artifactId>applications.api.identity.ch.gov.uk</artifactId>
+
+  <build>
+    <plugins>
+      <plugin>
+        <artifactId>spring-boot-maven-plugin</artifactId>
+        <configuration>
+          <layout>ZIP</layout>
+          <mainClass>uk.gov.ch.developer.docs.DocsWebApplication</mainClass>
```
Comment (ch-dclarke) [inline pom.xml]: Should this be the main class?

Comment (Zain-Abbas1) [inline pom.xml]: Is this correct?

Comment (CH-JonathanBlake) [reply pom.xml]: Changed.

Comment (CH-JonathanBlake) [reply pom.xml]: Changed

**Pull request merged**